{"source": "exploit-database", "filename": "platforms/linux/dos/40888.py", "cves": ["CVE-2016-6515"]}